docs/semgrep-supply-chain/dependency-search.md

```diff
@@ -3,19 +3,19 @@ slug: dependency-search
 append_help_link: true
 title: Dependency search
 hide_title: true
-description: "Search through all your dependencies in all your onboarded repositories at any time."
+description: "View and search through all your dependencies in all your onboarded repositories at any time."
 tags:
   - Semgrep Supply Chain
   - Semgrep AppSec Platform
 ---
 
-# Search for dependencies
+# View and search for dependencies
 
 :::info Prerequisite
 At least one repository that scans for dependencies through Semgrep Supply Chain. See [Scan third-party dependencies](/semgrep-supply-chain/getting-started).
```

Comment:
```suggestion
At least one project (repository) that scans for dependencies through Semgrep Supply Chain. See [Scan third-party dependencies](/semgrep-supply-chain/getting-started).
```

Since we are standardizing to **project** once a repository has been onboarded to Semgrep, we just need to remind people that a project is an onboarded repository.